packages/botonic-nlu/tests/preprocessor.test.ts

```diff
@@ -0,0 +1,98 @@
+import { DefaultNormalizer } from '../src/preprocessing-tools/normalizer';
+import { DefaultStemmer } from '../src/preprocessing-tools/stemmer';
+import { DefaultTokenizer } from '../src/preprocessing-tools/tokenizer';
+import { Preprocessor } from '../src/preprocessor';
+import * as CONSTANTS from '../src/constants';
+
+describe('Preprocessor Tools', () => {
+  test('Normalizing a sentence', () => {
+    const normalizer = new DefaultNormalizer();
+    const sentence = 'Today I am going to the office!';
+    const expectedOutput = 'today i am going to the office!';
+    expect(normalizer.normalize(sentence)).toEqual(expectedOutput);
```

Comment:
no need to modify it now, but in tests we should not use constants when they're only used once.
Read this https://www.notion.so/Unit-tests-8515f1d36352406ea461d06c08b2ee36#5b3d48716fac4ef6b76912f4e135a9e4